### DISCUSSION TITLE
Missing Macro for Nangate45 Ariane133?

### DISCUSSION THREAD

**Discussion by RaidToRadar** [The-OpenROAD-Project/OpenROAD-flow-scripts#1382] [Q&A]
After making the placement stage, I noticed that the design (Ariane133) was only showing a total count of 132 hard macros, is there any reason why this is not counting 133 macros?

**Comment by maliberty** [comment]
@ravi-varadarajan you added this design so please respond.

**Comment by maliberty** [comment]
@ravi-varadarajan I also noticed that it isn't running in the CI
